.github/workflows/codeql.yml

```diff
@@ -0,0 +1,69 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: CodeQL
+
+on:
+  push:
+    branches: ['master', '[0-9]?.[0-9]?.x']
+  pull_request:
+    branches: ['master', '[0-9]?.[0-9]?.x']
```

Comment:
This probably should just be `on: [ pull, pull_request ]` instead of limiting it to specific branches.

Reply:
Updated